app/assets/stylesheets/runtime-banner.scss

```diff
@@ -1,15 +1,74 @@
+@import 'variables';
+@import '_mixins';
+
 .runtime-banner {
```

Comment:
This entire `.scss` file can be improved/refactored. This is just what I was able to put together for it to "work" 😅